src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -50,6 +50,7 @@
     public static final String[] CORRESPONDENT_EVENTS = {"CORRESPONDENT_DELETED", "CORRESPONDENT_CREATED", "CORRESPONDENT_UPDATED"};
     public static final String[] EXTENSION_EVENTS = {"EXTENSION_APPLIED"};
     public static final String[] ALLOCATION_EVENTS = {"STAGE_ALLOCATED_TO_TEAM", "STAGE_CREATED", "STAGE_RECREATED", "STAGE_COMPLETED", "STAGE_ALLOCATED_TO_USER", "STAGE_UNALLOCATED_FROM_USER"};
+    private static final int maxCaseNoteSize = 32766;
```

Comment:
Maybe worth adding a comment? or a git commit message?

Reply:
Don't know if it's worth also making this `32,767` and renaming to `EXCEL_MAX_CELL_SIZE` and doing a -1 in the substring to make it clear?

Reply:
Good shout, changed to this and added tests